### ISSUE TITLE
Accessibility issues for new wp site

### ISSUE THREAD

**Issue by Klortho** [JATS4R/jats4r.github.io#34]

**Comment by ahamelers** [comment]
Here are the parts of the site design that have accessibility errors:

```
<div id="header-logo-image">
  <a href="http://seamerry.ca/" title="" rel="home"><img src="http://seamerry.ca/wp-content/uploads/2015/12/jats4r-logo-small.png" alt=""></a>
</div>
```

The image, which is also the title of the site, has no alt text. To a user accessing the site with a screen reader, this is an empty link, and the phrase "JATS4R" does not appear at the top of the site.

```
<!-- #header-logo-image-->
<div id="header-text">
  <h1 id="site-title">
    <a href="http://seamerry.ca/" title="" rel="home"></a>
  </h1>
    <h2 id="site-description">JATS For Reuse</h2>
<!-- #site-description-->
</div>
<!-- #header-text-->
```

Because the site title is in an image and not in the &lt;h1>, to a user who is accessing the site using a screen reader, this means the site has no title, and this is an empty (and thus invisible) link.

Because the logo image is really taking the place of the &lt;h1>, it would be best to combine these, either by moving the logo image inside the &lt;h1> and giving it alt text, or by replacing the logo with text in the &lt;h1> and styling that text to look like the image (I would be happy to help with this).

**Comment by Klortho** [comment]
Nice to hear you on the call today!
